2_correlation_analysis/spell_vs_counts_experiment/README.md

```diff
@@ -0,0 +1,23 @@
+# Compare SPELL vs counts correlation
+
+There are two possible correlation matrices we can use for our analysis:
+
+1. Correlation of the MR counts expression matrix
+2. Correlation of the MR counts that have been processed using SPELL
+
+**About the correlation matrices**
+
+The correlation of the counts matrix relates how similar a pair of genes are based on their expression profiles - **relates genes over samples**.
+High correlation means that a pair of genes have a similar expression profiles - i.e. similar levels of expression across samples/contexts, so genes both have low expression in the same samples and high expression in the same samples.
+* Pro: Easy to interpret
+* Con: Many gene pairs found to have a high correlation because many genes are related to the same pathway have the similar expression profiles. This is consistent with [Myers et al.](https://link.springer.com/article/10.1186/1471-2164-7-187), who found that there can be an over-representation of genes associated with the same pathway (i.e. a large fraction of gene pairs represent ribosomal relationships). This very prominent signal makes it difficult to detect other signals. Figure 1C demonstrates that a large fraction of gene pairs are ribosomal relationships - in the top 0.1% most co-expressed genes, 99% belong to the ribosome pathway. Furthermore, protein function prediction based on co-expression drop dramatically after removing the ribisome pathway (Figure 1A, B).
+
+To try to remove this very dominant global signal in the data. Here we are applying dimensionality reduction techniques in addition to scaling the data using a method called SPELL.
```

Comment:
This was sort of the same idea behind using ADAGE to construct networks in the generic-expression project, right? To remove the strong operon signal that tends to dominate co-expression networks?

I guess maybe you could do something similar here, but it seems like SPELL is getting at the same idea so it's probably fine.

Reply:
Yes that is the same idea here you're right. ADAGE would be another way to have approached this. I think SPELL was a simple first step especially for our downstream interpretations